cmd/dalc/main.go

```diff
@@ -3,22 +3,63 @@ package main
 import (
 	"os"
 
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/dalc/config"
+	"github.com/cosmos/cosmos-sdk/client/keys"
 	"github.com/spf13/cobra"
+	"github.com/tendermint/spm/cosmoscmd"
 )
 
 func main() {
 	root := rootCmd()
+
+	cosmoscmd.SetPrefixes(app.AccountAddressPrefix)
+
+	root.AddCommand(
+		keys.Commands(config.DefaultConfigPath),
```

Comment:
this is all it takes to add the `keys` command! now we can manage keys via the cli. obvi not suitable for mainnet, but definitely workable for devnet